src/logs/logger.ts

```diff
@@ -0,0 +1,38 @@
+import winston, { transports } from 'winston';
+// import * as fs from 'node:fs';
+// import path from 'node:path';
+
+// const logDir = '../../src/logs';
+
+// if (!fs.existsSync(logDir)) {
+//   fs.mkdirSync(logDir);
+// }
+
+const { combine, timestamp, json } = winston.format;
+const { printf, colorize, align } = winston.format;
+
+winston.loggers.add('basicLogger', {
```

Comment:
Do something like this. It's more maintainable on the long term. Add the rest properties as they currently are. It should work as before.

Update where logger was used accordingly

```suggestion
export const logger = winston.createLogger({